dissect/target/plugins/os/windows/network.py

```diff
@@ -238,6 +242,20 @@ def _get_config_value(key: RegistryKey, name: str, sep: str | None = None) -> se
     return {value}
 
 
+def _construct_interface(key: RegistryKey, ip_key: str, subnet_key: str) -> set[str]:
```

Comment:
Why not just return a string? It's only ever a set with just one element.

Reply:
I only just realized that's how `_extract_network_device_config` works... So confusing.

If you understand currently how it works, can you add some comments on why sets are used?

Reply:
Can you make a ticket for that?